apps/datahub/src/app/record/navigation-bar/navigation-bar.component.html

```diff
@@ -1,48 +1,111 @@
 <nav
-  class="mx-4 bg-primary flex justify-between py-2 px-4 text-sm rounded-lg text-white sm:space-x-8 sm:px-20 sm:py-5"
+  class="text-xs text-black bg-primary-lightest shadow-[0px_0px_5.1px_0px] shadow-primary-darkest"
 >
-  <button
-    type="button"
-    *ngFor="let l of anchorLinks"
-    gnUiAnchorLink="{{ l.anchor }}"
-    gnUiAnchorLinkDisabledClass="cursor-default !opacity-30 !hover:opacity-30"
-    class="uppercase font-medium tracking-wider hidden sm:block opacity-80 hover:opacity-100"
-    translate
-  >
-    {{ l.label }}
-  </button>
   <div
-    (click)="toggleMobileMenu()"
-    class="w-full flex justify-between cursor-pointer sm:hidden"
+    class="h-full container-lg mx-auto flex justify-between overflow-hidden"
+    [class]="
+      displayMobileMenu
+        ? 'items-start'
+        : 'items-center divide-x-2 divide-gray-300'
+    "
   >
     <div
-      class="shrink truncate"
-      [class]="displayMobileMenu ? 'block' : 'hidden'"
+      class="h-[60px] flex min-w-[60px] items-center justify-center lg:justify-end lg:pr-4"
+      [class]="displayMobileMenu ? 'border-r-2 border-gray-300' : ''"
     >
-      <div *ngFor="let l of anchorLinks">
-        <button
-          (click)="setActiveLabel($event.target)"
-          gnUiAnchorLink="{{ l.anchor }}"
-          gnUiAnchorLinkDisabledClass="cursor-default opacity-60"
-          class="block px-3 py-3 rounded-md font-medium uppercase tracking-wider w-full text-left truncate"
-          translate
-        >
-          {{ l.label }}
-        </button>
-      </div>
+      <gn-ui-button
+        data-cy="backButton"
+        (buttonClick)="back()"
+        type="outline"
+        [style.--gn-ui-button-rounded]="'8px'"
+        [style.--gn-ui-button-height]="'32px'"
+        [style.--gn-ui-button-width]="'40px'"
+        [style.--gn-ui-button-color]="black"
+        [style.--gn-ui-button-background]="'var(--color-primary-lightest)'"
+        [style.--gn-ui-button-bg-hover]="'var(--color-primary)'"
+        [style.--gn-ui-button-font-size]="'24px'"
+        [style.--gn-ui-button-padding]="'4px'"
+      >
+        <ng-icon name="matArrowBack"></ng-icon>
+      </gn-ui-button>
     </div>
     <button
-      class="block px-3 py-3 rounded-md font-medium uppercase tracking-wider shrink truncate sm:hidden"
-      [class]="displayMobileMenu ? 'hidden' : 'block'"
+      type="button"
+      *ngFor="let l of anchorLinks"
+      [attr.data-cy]="l.anchor"
+      gnUiAnchorLink="{{ l.anchor }}"
+      gnUiAnchorLinkDisabledClass="sm:hidden"
+      gnUiAnchorLinkInViewClass="!border-b-primary border-b-4"
```

Comment:
the bottom padding should be changed here as well, to avoid having the text "jump"

Reply:
Yes, good point, it's fixed now.